src/common/options.cc

```diff
@@ -3984,6 +3984,14 @@ std::vector<Option> get_global_options() {
     Option("bdev_async_discard", Option::TYPE_BOOL, Option::LEVEL_ADVANCED)
     .set_default(false)
     .set_description(""),
+    
+    Option("bdev_flock_retry_interval", Option::TYPE_FLOAT, Option::LEVEL_ADVANCED)
+    .set_default(0.1)
```

Comment:
May be make this default a bit higher, e.g. 1 sec?

Reply:
In my test, 0.1 seconds is enough, I don’t know how it is on your side.